account_followup/models/account_followup.py

```diff
@@ -0,0 +1,148 @@
+
+
+class AccountFollowupLine(models.Model):
+    _name = 'account_followup.followup.line'
+    _description = 'Follow-up Criteria'
+
+    name = fields.Char(
+        string='Follow-Up Action',
+        required=True,
+    )
+    sequence = fields.Integer(
+        string='Sequence',
+        help="Gives the sequence order when displaying a list of follow-up "
+             "lines.",
+    )
+    delay = fields.Integer(
+        string='Due Days',
+        required=True,
+        help="The number of days after the due date of the invoice to wait "
+             "before sending the reminder.  Could be negative if you want to "
+             "send a polite alert beforehand.",
+    )
+    followup_id = fields.Many2one(
+        comodel_name='account_followup.followup',
+        string='Follow Ups',
+        required=True,
+        ondelete="cascade",
+    )
+    description = fields.Text(
+        string='Printed Message',
+        translate=True,
+        default='_get_default_description',
+    )
+    send_email = fields.Boolean(
+        string='Send an Email',
+        help="When processing, it will send an email",
+        default=True,
+    )
+    send_letter = fields.Boolean(
+        string='Send a Letter',
+        help="When processing, it will print a letter",
+        default=True,
+    )
+    manual_action = fields.Boolean(
+        string='Manual Action',
+        help="When processing, it will set the manual action to be taken "
+             "for that customer.",
+    )
+    manual_action_note = fields.Text(
+        string='Action To Do',
+        placeholder="e.g. Give a phone call, check with others, ...",
+    )
+    manual_action_responsible_id = fields.Many2one(
+        comodel_name='res.users',
+        string='Assign a Responsible',
+        ondelete='set null',
+    )
+    email_template_id = fields.Many2one(
+        comodel_name='mail.template',
+        string='Email Template',
+        ondelete='set null',
+        default='_get_default_template',
+    )
+    _order = 'delay'
+    _sql_constraints = [
+        ('days_uniq',
+         'unique(followup_id, delay)',
+         'Days of the follow-up levels must be different')]
+
+    @api.model
+    def _get_default_description(self):
+        return """
+            Dear %(partner_name)s,
+
+            Exception made if there was a mistake of ours, it seems that the
+            following amount stays unpaid. Please, take appropriate measures
+            in order to carry out this payment in the next 8 days.
+
+            Would your payment have been carried out after this mail was sent,
+            please ignore this message. Do not hesitate to contact our
+            accounting department.
+
+            Best Regards,
+        """
+
+    @api.model
+    def _get_default_template(self):
+        try:
+            return self.env.ref(
+                'account_followup.email_template_account_followup_default')[1]
```

Comment:
You don't need the final `[1]`, and instead of try, just pass False as second argument and the method won't fail, but return that False.